app/assets/javascripts/discourse/widgets/post-menu.js.es6

```diff
@@ -301,6 +301,41 @@ registerButton("bookmark", attrs => {
   };
 });
 
+registerButton("bookmarkWithReminder", (attrs, state, siteSettings) => {
+  if (!attrs.canBookmark || !siteSettings.enable_bookmarks_with_reminders) {
+    return;
+  }
+
+  let className = "bookmark with-reminder";
+  let title = "bookmarks.not_bookmarked";
+  let titleOptions = {};
+
+  if (attrs.bookmarked_with_reminder) {
+    className += " bookmarked";
+
+    if (attrs.bookmark_reminder_at) {
+      let reminderAtDate = moment(attrs.bookmark_reminder_at).tz(
+        Discourse.currentUser.timezone
+      );
+      title = "bookmarks.created_with_reminder";
+      titleOptions = {
+        date: reminderAtDate.format(I18n.t("dates.long_with_year"))
+      };
+    } else {
+      title = "bookmarks.created";
+    }
+  }
+
+  return {
+    id: attrs.bookmarked_with_reminder ? "unbookmark" : "bookmark",
+    action: "toggleBookmarkWithReminder",
+    title: title,
```

Comment:
```suggestion
    title,
```

Reply:
I'm still not used to this ES6 property syntax, it's only been almost 5 years :roll_eyes: